index.js

```diff
@@ -220,16 +220,19 @@ module.exports.spawnPythonService = (additionalEnv = {}) => {
 
 // private
 
-function findBestSubtitleFile(list) {
+function findBestSubtitleFile(list, locales = []) {
+  // favor locales but some subtitles just have the language code
+  const localesAndLanguages = [...new Set([...locales.map((locale) => locale.toLowerCase()), ...locales.map((locale) => locale.substring(0, 2))])];
```

Comment:
Pretty minor but just wondering if we want to preserve the order of the locales? I think it's been implemented as first one is the priority in some situations